packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -426,6 +429,94 @@ class TimeInput extends React.Component {
     }
   }
 
+  handleInputKeyDown(event, inputType) {
+    const { second } = this.state;
+    let {
+      hour,
+      minute,
+      meridiem,
+    } = this.state;
+    const variant = TimeUtil.getVariantFromLocale(this.props);
+
+    if (event.keyCode === KeyCode.KEY_N) {
+      const currentTime = this.getCurrentTime();
+      this.setTime(event, currentTime.hour, currentTime.minute, currentTime.second, currentTime.meridiem);
+      return;
+    } if (event.keyCode === KeyCode.KEY_DASH) {
```

Comment:
Can we put all these `ifs` on new lines

Reply:
Went ahead and put the ifs on new lines, also gave them new lines to separate them a bit more.
https://github.com/cerner/terra-framework/pull/1503/commits/aef1fe13be57204c740370d8c4090c417fb8515d#diff-37c08a2b0d74d4e14dd7fc4e7e160bb6ac190dd983aebaaf30397c71621b735dR445